Dockerfile.development

```diff
@@ -1,5 +1,7 @@
 FROM dependabot/dependabot-core
 
+RUN touch /etc/dependabot-core-dev
```

Comment:
This means the script effectively requires a development container specifically. I'm not sure if there's an argument to put this fingerprint in the base image so the dry run script can be used in production images, so it seemed best to be specific for now.

Reply:
Pretty sure the script is not included in the prod image 👍